docs/cookbook/creation-of-custom-code-sniff.md

```diff
@@ -0,0 +1,120 @@
+# Creation of custom code sniff
```

Comment:
I do not think we should have a cookbook for something that is already present in the code. I can imagine article "How to create a custom sniff" that would be a part of `coding-standards` package (linked from `README`, see https://github.com/shopsys/coding-standards#custom-rules - your new sniff should be described here as well). The article then could describe the implementation in more general way and for examples, link to the actual implementation. (We can discuss it personally :wink: )